src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8406,5 +8406,49 @@ class MyAttribute : System.Attribute
 
             CompileAndVerify(source, symbolValidator: attributeValidator);
         }
+
+        [Fact, WorkItem(10639, "https://github.com/dotnet/roslyn/issues/10639")]
+        public void UsingStaticDirectiveDoesNotIgnoreObsoleteAttribute()
```

Comment:
>UsingStaticDirectiveDoesNotIgnoreObsoleteAttribute [](start = 20, length = 50)

It feels like this test doesn't belong in this file. Perhaps it should be in src\Compilers\CSharp\Test\Semantic\Semantics\ImportsTests.cs